cloud-foundry/nginx/conf/includes/redirect.conf

```diff
@@ -0,0 +1,3 @@
+rewrite ^/starters/task/maven/elston-ga$ https://repo.spring.io/libs-release/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Elston.RELEASE/spring-cloud-task-app-descriptor-Elston.RELEASE.task-apps-maven redirect;
+rewrite ^/starters/task/docker/elston-ga$ https://repo.spring.io/libs-release/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Elston.RELEASE/spring-cloud-task-app-descriptor-Elston.RELEASE.task-apps-docker redirect;
```

Comment:
Need redirecdts for

```
https://dataflow.spring.io/rabbitmq-maven-latest

https://dataflow.spring.io/rabbitmq-docker-latest

https://dataflow.spring.io/kafka-maven-latest

https://dataflow.spring.io/kafka-docker-latest

https://dataflow.spring.io/task-maven-latest

https://dataflow.spring.io/task-docker-latest
```

and we can just copy over the current bit.ly url format , chaning only the domain.

https://dataflow.spring.io/Darwin-SR3-stream-applications-kafka-docker

https://dataflow.spring.io/Darwin-BUILD-SNAPSHOT-stream-applications-kafka-docker

etc...

https://dataflow.spring.io/Elston-GA-task-applications-maven

Reply:
Done.